test/Makefile

```diff
@@ -0,0 +1,28 @@
+# http://clarkgrubb.com/makefile-style-guide
+
+MAKEFLAGS += --warn-undefined-variables --no-print-directory
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := test
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+tag = claranet/direnv-asdf
+
+.PHONY: test
+test:
+	mkdir -p .direnv .docker/.direnv .docker/home
```

Comment:
Create these here so that Docker doesn't create them as the root user when setting up the mounts.